mesonbuild/mconf.py

```diff
@@ -47,7 +46,7 @@ def add_arguments(parser: 'argparse.ArgumentParser') -> None:
                         help='Clear cached state (e.g. found dependencies)')
     parser.add_argument('--no-pager', action='store_false', dest='pager',
                         help='Do not redirect output to a pager')
-    parser.add_argument('-U', action='append', dest='unset_opts', default=[],
+    parser.add_argument('-U', action=coredata.KeyNoneAction, dest='cmd_line_options', default=[],
```

Comment:
Shouldn't the default be a dict here as well?

Reply:
I think the default is not used. Will check argparse sources.